torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -696,8 +718,219 @@ bool isFlattenedMul(Val* x) {
   return toFlattenedMul(x->definition()) != nullptr;
 }
 
+std::pair<int64_t, std::list<Val*>> getConstAndSymbolicFactors(Val* x) {
```

Comment:
Could you add some descriptions to your functions (even if it's just a sentence)

Reply:
added